src/steps/audio-setup/mic-preview.tsx

```diff
@@ -144,6 +146,10 @@ const AudioVisualziation: React.FC<AudioVisualziationProps> = ({ stream }) => {
         flex: "1 0 70px",
         backgroundColor: "rgba(0,0,0,0.8)",
         borderRadius: "7px",
+        ...isHighContrast && {
+          backgroundColor: "rgba(0,0,0,1)",
```

Comment:
That's just ... `"black"` right? I mean nothing wrong with writing it like this, I just have never seen the use of `rgba` if the opacity is 1. This I would expect as `"black"` or `#000`. But :shrug: